src/lammpsio/data.py

```diff
@@ -267,6 +347,14 @@ def read(self):
                     box_bounds[2], box_bounds[5] = [float(x) for x in line.split()[:2]]
                 elif "xy xz yz" in line:
                     box_tilt = [float(x) for x in line.split()[:3]]
+                elif "bonds" in line:
+                    N_bonds = int(line.split()[0])
+                elif "angles" in line:
+                    N_angles = int(line.split()[0])
+                elif "dihedrals" in line:
+                    N_dihedrals = int(line.split()[0])
+                elif "impropers" in line:
+                    N_impropers = int(line.split()[0])
```

Comment:
Add the bond types, etc. headers here. Use these to do a sanity check like is done below for the atom types.